File: 4_go/EsPalindromo.go
package main
import "fmt"
import "strings"
func EsPalindromo(palabra string) bool {
  base := strings.ToLower(palabra)
  resultado := []byte{}
  for i := len(base) - 1; i >= 0; i-- {
    resultado = append(resultado, base[i])
  }
  return base == string(resultado)
}

func main() {
  fmt.Println(EsPalindromo("mex"))
  fmt.Println(EsPalindromo("Lol"))
}

File: 5_typescript/esPalindromo.js
"use strict";
exports.__esModule = true;
exports.esPalindromo = void 0;
function esPalindromo(palabra) {
    var invertida = [];
    for (var i = palabra.length - 1; i >= 0; i--) {
        invertida.push(palabra[i].toLowerCase());
    }
    return palabra.toLowerCase() === invertida.join("");
}
exports.esPalindromo = esPalindromo;
console.log(esPalindromo("mex"));
console.log(esPalindromo("Lol"));
